tests/acceptance/features/apiSpaces/tusUpload.feature

```diff
@@ -0,0 +1,26 @@
+@api @skipOnOcV10
```

Comment:
Why do we need `@skipOnOcV10` tag for the local Ocis test?

Reply:
https://github.com/owncloud/ocis/pull/4126#discussion_r920812065